resources/var/www/html/errors/starting.html

```diff
@@ -1,29 +1,31 @@
 <!DOCTYPE html>
 <html>
```

Comment:
Like in cloudogu/nginx#103:  When we activly translate all text we also should support screen readers by setting the page language, shouldn't we?
```suggestion
<html lang="de">
```